src/auth/auth.module.ts

```diff
@@ -0,0 +1,27 @@
+import { Module } from '@nestjs/common';
+import { JwtModule, JwtService } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { RedisModule } from '@liaoliaots/nestjs-redis';
+import { Env } from 'src/env';
+import { JwtStrategy } from './jwt.strategy';
+import { GithubModule } from 'src/github/github.module';
+
+@Module({
+  imports: [
+    GithubModule,
+    PassportModule,
+    JwtModule.register({
+      secret: Env.JWT.AUTH_TOKEN,
+      signOptions: { expiresIn: Env.JWT.EXPIRES_IN },
+    }),
+    RedisModule.forRoot({
+      config: { host: Env.REDIS.HOST, port: Env.REDIS.PORT },
+    }),
```

Comment:
PullProofAI

The Redis module configuration should be updated to use the correct environment variables.
```typescript
-       config: { host: Env.REDIS.HOST, port: Env.REDIS.PORT },
+       config: { host: process.env.REDIS_HOST, port: parseInt(process.env.REDIS_PORT) },
```

#### Overall Comments:
The `AuthModule` seems well-structured with necessary imports, providers, exports, and controllers. Ensure that all environment variables are correctly accessed and used throughout the module for consistency and proper configuration handling.


<!-- This is an auto-generated comment by PullProofAI -->